From 21bb88c404de12902440c0b83ba7e80be99cd434 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 08:28:34 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh armored-keys.asc from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 armored-keys.asc | 639 ++++++++++++++++++++++++-----------------------
 poppler.spec     |   5 +-
 2 files changed, 333 insertions(+), 311 deletions(-)

diff --git a/armored-keys.asc b/armored-keys.asc
index 85382d4..a73c370 100644
--- a/armored-keys.asc
+++ b/armored-keys.asc
@@ -1,6 +1,4 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
-Comment: Hostname: pgp.surf.nl
-Version: Hockeypuck 2.1.2
 
 xsFNBFfNZD4BEACo0YKCUEhTY7v8VZVw3csAtnsRSFQ18G8xfYIt7mJEBrtfiGG4
 g28EjNeWQfN8wGIaAeRGl5nb7s9RRpxXStUu719jGX1Z9Sr8p/JOmBS5kdfAtYId
@@ -13,12 +11,32 @@ B2SrSj3ImM5FO0XGd1gxn/qywNVJDQBfcYnndksD2i/9/Es8I/F6OymBZl4HWBvx
 P0B3l1E4Ccw1ne2/O1BdsnXeUaI3YVYz9aZkVlL7ywda3h/goRKxc1b18bmgu/Ed
 QmcTEMj29B3szMpetWNt1ZbaMJzfDw+z+SiQ3toOTUiqMuWq+l+JK0dVUzOvpGhG
 VHv5m9CtJFP671ivEc4it0hJRH5kdW9aNbeG0g9TxHQqQMJIpeLvRq5WKwARAQAB
-wnUEEBYKAB0WIQTpP9uBhK4kHDRc4jXCH/n9yHqpbwUCW3MlTQAKCRDCH/n9yHqp
-b2O9AP0cOqSP/7RPLpOfZ9ck5AmrjGtcpA8H2YlAqaOtpOmr1gD+OOul6zMFOwHu
-xRid7Q9ro1wsBmbn+NgKDMd0efLOXwPCdQQQFggAHRYhBFMiBEFRHK5DvalSBn+D
-wv2ENls4BQJbenmiAAoJEH+Dwv2ENls4GzIA/3OnVVGjBwlL0MqjeMynWrznsbfq
-mcuttMTeW3f0UPeaAQDi8WQRYPXfQF6UG38OoXiA4Bi+PHb+9tcBW10m6PxiCM0h
-QWxiZXJ0IEFzdGFscyBDaWQgPGFhY2lkQGtkZS5vcmc+wsGUBBMBCAA+AhsDBQsJ
+zSFBbGJlcnQgQXN0YWxzIENpZCA8YWFjaWRAa2RlLm9yZz7CwZQEEwEIAD4CGwMF
+CwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQTKJixsg95NL7KKMyo6ak24Oeqm1wUC
+aeaYuwUJFLL5/QAKCRA6ak24Oeqm14ZgD/9rFTlhsL2ynepmOfRzdmUTBbxYQeFU
+7YS1RlJX1nr5AlGFj55GUZekIGm5XB85ZHBDPkfHjUXneSkV9vNwQyWkm2PsZ5yq
+0bNO0cHykwwqmYvQ9A/nn7WHT1/Zxs3JYaiGLyqX7q/+LjL8mbDjfn/r4Ey9+/3P
+WEdLyFEyYVwzLuY+kssAcThdJ6kg3MgqgfRTcjoIfgMdtoBRB2lnvqmf99Hd0vhY
+MPUaKfvRxxfL/5wEOSKz86TGZC/z0+GuZZGouRehLETBfvIolcmp/ND4JB0nHFqq
+3djSqc3M2DOF9uEmpNq5j6FA4uu5cLSN74in7V/OkEb0PyS4aQI0Gh4/uRN+Ww0o
+qWLParavtK//XuFueI06jiWf4mmpKqsq3Hh77bbuGg0E/Rc68cXqU7HPxjzmrafi
+rsqAmGZrEc4uKj87MiHqe9HJEPNwfqPqfrCMhl4KWmK0/3J1fcQWU+k3nRB2unhl
+d+hnyhFKU8zT9LN98Vvjdb46Tgy0NDPEzc35Yb84sdS33r8SlmzvID6IfOftD60p
+LIxems64nLrXgXxkGhzsQT71RtdlubTITA0CBQq7aexLdYOJewiyCZz9rycq7l1U
+U2zBMGZb21/BR+zqzpiGCgy9+YAU43YgxiXPWCTZH+aYpF6qIAA6/QGHck9nvd+/
+v5RJaETw3Hb3ksLBlAQTAQgAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYh
+BMomLGyD3k0vsoozKjpqTbg56qbXBQJk+j8IBQkS0c9lAAoJEDpqTbg56qbXx1QQ
+AJ4PY6AOAvUJMyQMfp06GoTxNq+DnkGI3OGPshsYDRtoPkVRWDjA2yI9fxj9BhfG
+gYNA0fz22n/pyXBC2OWRuW+IZJ3YL6xFfHP1lOVt7bLMNLOGyvy+TJF0zRvuA5vF
+HZ4cWbF1kQa5pX1Hv16JjoFx7Y0U8n/XVjXcKr7QJ1zS9/M5l90LIMV54AWlmC2c
+zY5cgpSEHdSmjEtZYI+F+ezYFfAe/frVgQgwsALHDhcLKOUh0qzu4mwRdpUoQuNP
+hhMOSmz183csvTJUm12/+eNEkL3z7mD1PUf02bn2SII/5USS/gePz0FRTE73CGhh
+pzBfSWKEBfRO4IAHme3bUv7OTi5ZCve0sGR6va429Zpsc5NiWVeTbJ6IdONgBTIR
+c0PyYr1o0iriCLf9z1CPLSJu2KH5hdxMO7lwDGFCtBrE/sIvZxaNAGWYkcmruLN4
+sWgSy1tuQy2IColGo4mymFRAxjdxqSAB+IU2q6+yi9utPHmmqQvpcqFPoLVcQHpt
+HmRqeUHu8VcXCUkmh7sSb/EPpvVE25JtL/K4eHuMgJ1W01ton87w50qMNprlKZTQ
+jOM+sNad3dg8zvpv/uTbXW8Uv7AfooJsjNnbZCF1Cx1pAKG0vGr2IJm4e+szeRQq
+iJWeKBg4/W/eRdlXoM06xcMsZ+Vp8C0pHZAkDMr4f0JswsGUBBMBCAA+AhsDBQsJ
 CAcCBhUICQoLAgQWAgMBAh4BAheAFiEEyiYsbIPeTS+yijMqOmpNuDnqptcFAmL+
 dv8FCQ0T1yAACgkQOmpNuDnqptc5LA//eR3bC2wjJUFiJn7cQGNMFDVS077e/va4
 6bf7p8gqnbQ/bkuyek35XZndsz8SCRcaQ8dwCuYCRgJfIQbgEQUxupsw0sqIctJX
@@ -105,143 +123,90 @@ PxXNa96Qde/79btqHlcMJB6gT0FHuXPx1LcyAlmtFX9JcZ7Gqzi1TJxHpjkUZTgo
 MXEwyImKqVKX0O4rkOvmHconeiQmg5+bmzi7wY33OjFspWjSpBWZ+y4D9g0orPAo
 t8Puzhrcg5ZSfRr1nZoQuk+IllHbO7xbnQOAyhNBIjRV43HXDG1xzRpm1pvxAXRi
 rFrYAAVxyaMDJ+yAZcqdoQZrVRF6+/qQQP34N5tVB9kKmEFMnxsCKVUQ7eu8RDIP
-leea2j84TajpniwflYg+V0cw+z75mSq3FV0lwkYEEBECAAYFAlfNZ0kACgkQ/lRE
-vmcCFhuUaACfXXiXGmBB7omAJoaaAIijN4RvaJEAoIungRgZitCTfoJxmUDo/uHR
-vlgOwsFcBBABCAAGBQJXzWjHAAoJEC6zhAEutxNY/K8P/3vKpKnOw5gjlyNcDJZ9
-F05Lxxkw8ZWuBqmPsOrgoubV+/mp/ZZl3EqT3WM1/RCLblxYXqAwGMNkhsso8/QZ
-ujhRjk2Qy2SgIN0v+768ggburyaBQ8kXzKpW1ZJKVK9hPSH+WY9Ym5HNP7RTUdwq
-nc0ut5+kRL6qNUM9rjdKTQsX0FT6oBPy12GiGFdJ1aGxUMX6WuBas8UvzLDMfTsZ
-JtktkC3GsbOoCqGWlfpIu70lz/tqfvR06ZElZWjR7uTyaiANP8Kv55ouYhaTtfTZ
-MtU5UP7mNNi7f2gmNtGd7Y29AcXdfX9TQ7tBpXEzI/JnuItWV/y0GoZv11k0tQPh
-X7XZYJaHaIe6daLTloNdCMv8l1pfPSziRaNsGapQJz7iL0oglXqx8DPvX1DPON3A
-Njgz9lzw1YPvWXANCMjEsnk4aXOoGOQWJZ5/rEpigKWgOT4gMoF+VqYEpnGbthUY
-K/KVWOjJaVxNMP3+61LVZQZaQgmRuYf3DuBmM29bMLOdWbCfbSqfpHbHbwDBCQdW
-iz+Pwt4PezgeL0Jg5MWmnh0kDaqor69Ry+Xb12e1n1NI2CMNCSy+QzpdMOGwba6H
-IHNYGmmFaxErO2LmkhmMFMNcP5rjrVDr8TXJDUYppLCvgG+gnFFJW5086rS29pKC
-iGAkoMEyEKbeBnRuDlS8ZgGBwsFcBBABCAAGBQJXzWo5AAoJEH/qPaYWnHfWDi0P
-/ReX2QDGsvdXu6fNFBPussV9diJgHzfTKjvMiagdeMfL+pav0qZNODKrAz8EYO4z
-OqY0wn55iY+5NmWqG5ynhfo8epnRdJK6Zf2ZEfV1ABDotKxyAhWfH61smF5bjC6v
-XAVv3dHCP/WNJwSYem5bO17e0ouofGjp2TLVqFgynhfGYJZdy9ay2s9qFLx0o1Xk
-K33hmp6XXmRnuMoM3VuM7RaSaIRFSFVdg1+eedbmF8whxlExiBNKii1q4yiEdczf
-LdrRlJHhC8mFnv/lZ4OxA0OAZBRY0KMcqL/1/6j3d3uFJfUoPvtjquNyMefhRLFH
-kNQIID8Rhlw8QfQXzLw2H1j40iU2RXuV75uZC8QQwUtAHdFqdTtGXFeleZRln3BN
-IKcFpR4ij3o6qId5CW/+BwTDwRlwA1+fxK+NEcIH+Iy4n2P87CscEJQPmNXxicqO
-hk7rpM+5U3tXr4Qgu2Rz2YuTGnoaOxzIzQ0jQQM54dZOkZz05e0zBhjNWAgPrVdZ
-IqXy2LVhbPvKPfO1YyuGJf3A+WbKnXI1TdqPSbtYcBQQisVN6RRcc9bxwsPHioh9
-TC7wllpX3tB1v6DJm7UD4kcGuTErzqmCya+VBjQ9/+vWiP35ixEvbptp8BO9/qEL
-wQ9PX3zPSPvGfKpdvBxPnURO5fl7WBvDzxo/Qxxca5/ywsFcBBABCAAGBQJXzXEY
-AAoJEP8lURlvuP2A3C0QAK8u3I10dQurDNvzGU/J3yJW5UAlzSkUmm+hWRk86AzP
-O6PTz6g9In9AWBQhAfq677bqLDbaVI77Vr56+UX8iGU3xD03mhBzlCQEFaAsxncL
-GjF1s/9sd7rOw268WI63a56YkHPCBoOxnVwnFAzrya/UUQ6Fhqh4V+gaz2YQCgRZ
-V58Sqi3YwyW617eNSb2u/zwQe3KkUB9+ydve0cQO5sqpX1DLWsGYqawm03kzCWkA
-SuTaFMMdiZATypm14GZY1EPcTZ6sI8ZfjQ1JQgWNSOZLnGT8q5Qh+3DXvJXjVbaa
-EHd+SABEZZnhgoMbPVDEX6Wh91pVWQoMHc+yZpDP2961VSMdHCa7DAlOPN7Vp1q2
-5fTbQHzMR+b1/BwuorJieamxNSNz9b0X2mI05sPR/fXlUco/4ljHuNVgqXBDxCIt
-AUYwWs1Evxp0WXfSYcZ40nZCkdLYlQHi5ciPF1wZk9DGx27qSVkn+bSarSoGwTfn
-vzqE22W5wflOpCmf8oUXXpkETYKD3OB4vpC2YFLrL9QnI0y++ro6x3qvpvgZp4o1
-mmR8IVlQDXaZtCtKAoj/5vI+n5G8WEvCGH3/eUpyBcVSIUUgJOONjQhHbcMDOYs/
-l0Rat7CpK35H/hnibwKDf5Kh7o/HWyQa0zu1E/+cIHRDh5qup3p9CNaoLBz59S2Q
-wkYEEBECAAYFAlfNnNoACgkQ72KcVAmwbhCIQQCffZzNo3T0XlboGMRJEm9OI4p3
-uuMAnjVoiYj2xZP3ftUXpyUYe7pYSuk4wsBcBBABCAAGBQJXzZ3QAAoJEFjQ7mSK
-SLO7U9MIALx6GhTzB7m8LyuGDWk7Jq3VJQIiCTXy9YHIvo6lo5WNwpTvRvWww8mF
-qXOXJLt561cp5eJ3fGZV+RBH+YYH5Ju3j1yVzLBf695yWTUAFwFG0dHLoUN138Mq
-mRr6YmpXIqZ6rSYLlonXzOO1r+dLJhGAbXqK8E3M7pBveYuG+Ssej0iQhdnXgBly
-b2KD8oNKXCzAdi2P50SR/9Y334t5Vb60Y+DokJpNd2xJvwIa0I1McYiDH1dyyECA
-fzN9/oXTljP1eIdNFAWdBVLQiog3hVf+iUGllwJwGPbEw/m7EpBMk5BF7ZQmrGtB
-pszAA3uH6jG3L+D/jEi8WE/YbXskZJzCwHMEEAEIAB0WIQTwfYXKoYrPRqNG/QF8
-f8bqhjO06gUCWP0YZgAKCRB8f8bqhjO06jXfB/0Q68/0kwjGACjI7cuyPTjmEWK/
-DEQgn7SoSBnjO0BG6BO1jeXNx46EoO5b2bbCO6E5OaW6iZPBomRlpJqYJS04lEy4
-WAadiVT0np1QII6nmMlNmh0ue0Ythj0qDNre4kriv/Lfq4HU5YVqcKUWG7SA5mPh
-jA0CuTuqojNMDC7bukv/slRp40FmyTGLMzkyZVayQi6cmcUxQI0HQQznj1/j2J1F
-v585pPCv+F5E3H8UGbN88OvuoO+x99k/48A8b8cqdWDYfvuLvyAGpbx3/ecpazR/
-AyUfH1AwvCFBHAYR9Tr7z/s01rrJg2pqCjtTCpTLFcl7EHVX2OCGn9uvqhItwsBz
-BBABCAAdFiEE8jJ15L8Qr8HfaRSm29LOiT4tHIcFAllkAEQACgkQ29LOiT4tHIec
-8Qf/TCJVBbwLn5txHZfzjMDX5YzTKiAakOTYriEhIYZPNcsRifAmlSZzTexHfyOD
-L1X/FESaPMepHsk3/AzvEZvBQGPnoaolKtg9M0inlYz2cmuFoZJwD4RKieKlh4Y9
-os97Qq2v63FLgce6CABTQvcyBkOSKgDpmVnDHj7bd6q8jb6P33JVw7+CGigl8M5K
-PNAvt0bh74LBDfti/4UJnPiXkujSc2cEgFSdy71lrutxcGN87NfM/LSOxUDtXgTi
-5Ag3GN4WCSS68/FfD9svT4kOEHpZCqbDU9hTFH/Qs3qJhTbvKRbaWJ3NLt01Xdyi
-HzpTV45Ax9u8mEtTmjalyw2g78LAcwQQAQgAHRYhBNL6q2IfYY1CaFtCe2PXJkwF
-aH1+BQJZd2ptAAoJEGPXJkwFaH1+4UUIAKgBUVlDssQClidllN7gY4YS6Xo7HE2D
-P+EM4X7bWDV02tTIZx2iskN7uEsb6/Swp2YjgRCY2fu7MDrna0ht1M/Ca+/78nq3
-+7i7vHhxVtbfuIUOi8W/eIU+XgjyguEA9Ahs18aAYWXQreWZ/DJKFJIJzbMtUWfH
-Wc7sCYazm4xx5fUxkCeI4jy+VBkRSSqdEiCM6hFPLs6f360V0kyEg2IGZitVXZOU
-rcy2WP/wzMPJjDEgzpQ/2rHPVoStQbqNFYVVz4mZcVqLoW+8XKV5NVt8lMTnHYTP
-ORoiP8ARZMjUWFMTolLBgDHkHYvGmi4x23yFYDPUZVrjBtSak4fcC9TCwXMEEAEI
-AB0WIQQZlJDd7Iq1vjrsvD6M2wAjAHol3QUCWXdtHwAKCRCM2wAjAHol3a24D/9t
-3tXbqN+V6BqrTURo2VT6CIRMc10Vis4NNu7ifsMi71h9cFxuiZi/w8y//Ckc3sVc
-WfrqxomjgtLwGccle6WR1ps+LoOsBAnUkwEPI9W7QEg37DazLDVDPSU9/dxYh8uQ
-tySctwaMrE6le5WeQ635jsG1UVz/P00thEzDIZpzlP3hfgE8+mS8O1kRxufAdv5H
-piHMuTVVyVVo+PL9c/0k1ORIkdNlbjFDBO7WVwVZdJtfRj4gHuwLI45TO/fzBdYM
-AHIO8RaEonS28Z8haxDOupDzbCbOE1JEvTWfmdoCleGuoc3t6VHOrcSzYrVNl7sR
-pFlKRjmdBYq7tKmBsLm4IJXetGNUVERKaH7IHP2K6r3IWgU776sme2OQlhAeW++q
-BHhto7U+tEDYeCLawnSKNCAVSw9GeXI4RYKu1yBndJ64wGEp23yITcrzfbMs03fq
-j5gGXrr+XUtTWjYb9MwkqRVj50VSAasEcfiPJKStPo1LbBp8ga3eVFjDqtBtmbNF
-VDC3fjH1cG9R34UuD7tkFMLjBuirgkL5JSgQ5fKNryYNeSUgLdMDdGrlNwyIN34S
-d1dNwt8FlGZA+qjThX4uvcoNqisnqqujKPULt9XmVDApyCzcqOj5tHwkQezNF/K7
-7SZ8gcOQpVXhSPo+vYgnowlTbDvOPZsbADGkwlkV0cLAcwQQAQgAHRYhBAqsd1u2
-Q3qNmvejrP4HhBF/vOEdBQJZfrDKAAoJEP4HhBF/vOEdlFoH/2BXeaZA/Fui6SGe
-OjoPaOUNkrl7+onQiCeNH07lVE6IW0+Ud7XUBeNjNxr2tIsB8q4uI8XW3MYQLnNb
-ENtXiBD59yfwu3yIcm8t6LUdeG31PUPI0xTeeOAC9Wakg8+aHKpiKGgwhz2eUlcG
-VB0BS7hNms87DFcjW+NqcKi0daGgdv9GWW2wXKxQ8hzA6j7WIWsEUwA/7Rx34jDA
-gwjGDr2sYbBT3Z7R+mi0F5F1WfVVIqADpBeNYIRGNzkurdkAy0tM7OQKDRJTpfVc
-Vv/hWB8moNmlvRX/+yuJIbmRylOI3z9lMTEV81MGNr8kJ65AqUCiTdoLoqUsAbcN
-0tD2O6HCwVwEEAEKAAYFAll+69sACgkQP9u1UITMXYQd8w/9HhQUC36+B5QGSCDz
-ZKHuWrxiADGiw0thtSP9xn6+uvrpKpa5HaBNBVAqxKVLTJ/S+3PpvAoDlb8Mw3bH
-umwbHisxbCOK+vviLSHHLXC29LXE2IjRihQnC4WsqytDwarHTT7t/AkFvmvtddB3
-356KbCnEP8KjS7KRL50/DkDqKZLJb8Sow9VQnfuGXwDo3gCEuMZVyD6IpFRizHCi
-jpZ6iEs0x7HHgcggbBK/n49X1KAv91ycTO+l50Bd83ChCmC0hmCbmStgQs26ZaxB
-yEI9fhnsqh7zcV7VkyfLGw5KxgKx1KEUIHTCyqyRdcvGRIRA1RWX4tsOjSZjESsW
-lDDh2kjhbjjeVf2VSD6oaY14TqtjNRucmDPYMdmdFMK46jk7eROm07sE/N9XwGSF
-oKH5fB/iTkZN0UkRNz2DQiYlCofYInrR1FmuM62aqzfLnoM+LRLc9o8XDG+ILvZq
-KG05oHBf3Hdbr9dfQ5yD+2Bi5YqWoKtUJfOiGS/iJ6P090dJvIi6xM+OgF0SZmkY
-Uw6b/5+F+hkKorxoPnaMSEDyOcrzlCxtLiTu9yk5GTgR8oKQlGjO2cDS7SBQrN4f
-Dh3ixgM9KSWnT9OSzb1uGXZzY+IiC6Bm+U5M8MSmo+LbQaT/dnfx/6ywvVAwGXXI
-y39kZG+ld1jMYBwmUkiWDms/NEjCwFwEEAEIAAYFAll/P3kACgkQ7JTRj38FmX7T
-tQgAqq07Lo9Jk58D4D14EqbkHOumvmsYbjkwWiYLuBV6zX2PaSvVFO/o1qo+0eII
-cFWPO3ji6fDMYBhl2tuub4hKE0DbKzdCDzfMrU/SiYSWo8ep3KrXp4lOi3yFJCz2
-WIhi/v/5rz9wFZ/fBufYVjIPKYXzuZTISciy2r2z9X5jx1wTO2doW2WBf/RFCApF
-0oUBBZY9msFJbmA7HU/2ifl3ia8CJLr2dNLFeFeLINjqytLfKwcVQHV3GOWNbF3N
-ZnkJ9svw3ujQfix7GvcaLILKz5B+0BWFVmA8BIGic63xghrYk7wMJHmyzsPMitU4
-OSN58QwSEZAgHdOm+2hJ7MykO8LAXAQQAQIABgUCWYqtlAAKCRA1iNpBPJiXm3/n
-B/9OBDmBeQMuYNSs9CdsE08ptPl3R30YNJITIjkJYyU/wfYirlQZWbiUxuplcbfW
-h12RfxvqJOJLXlL1bH1l3fKnPWDaoa3DXyz5qPZmYRVEpHsezHqekBHibGru2yuG
-N0w6owRbIVr7H2vAV4HboRm+5AhH+Y0EOhMF1+S5srhUcNhyR7rXjAV/fFOLykpx
-RVfEt7HSdY0WAs+oocucEXf0sHvV+J4t5x53NWOejb1URUxSsdfx5WZHK4NsFzIw
-RuZt4CtAEcz6BM5S7rNeIuo1UAJV+ipNglbFNssU58seaInhE76kSjAW8MzhCOxP
-oDipDxWUx5uWZjAm8aDm5XwQwsFzBBABCAAdFiEE5e16J9/c84GC9eQ3EnD6a31E
-ZocFAlmbV9wACgkQEnD6a31EZofIpQ//fEjizSo3Nvz1G1SIc8zoV4dG4nBkvOUC
-StgxXEj8iTZIKIXjdGpwVcGOCOHTak557u3E3MSbp+DHwskyv9ShQIti58k7e4Tj
-NxnyFDNo152nPB0wXVrJytsCWfXXesgpST8+j4qXntdVKlcu94fwOvXf/trPqkik
-TrneyzTv5gv4q4MkDkwo7piwPitZ6hzpM2kRlUkEZY6tX5tLMtKnYYTFLBf8TuSa
-x3bH3TTcUdLPJEdVhRtqbE6mnwiBJxkPgUTRPmfKvmq/061gEolmNmpyjHXQNwM0
-yVTdpTgX1fsZVBsk7CLvGCdkQ216iA5x1tsmYlWhjy3g51wXpZPRL9PuXo4mZW4T
-+C7UE3KsZpffIU+rWFDynutjUzJ3hXldcH/IgSVEbiDxKvO1eGaossYOTzZVmxYE
-ZDE6uFfWWrDhR32sMlBhz0aKGNGCVuaVJVgxkCmP4rOeR0U3QeaMaX2LFg+TxuOz
-0V+1otBMj1xDBOkz4yQJJdUQfGgN6/XSLYYivXWOFt6b1CX04yRWFlnz2xhHuyxC
-31iQpcv1+rVKPFVICEDUwY4d2sBT2Up3NN+5VrBUjGf8i+o0Gjg9s2n9Y2eUVnuP
-lhTHJbTLKwHJRuk4p6tUAieWuC3KO3s7uvL/nqtdBJn/GUb0UkgJx6eeSaS2pprL
-abr/pPPkytPCwXMEEAEKAB0WIQRCsMoADCWgBsl6oOR4l6AmJ0QtlAUCW3MlSAAK
-CRB4l6AmJ0QtlBhHD/9B7YmxAWXblPuzfR0O/I+lRj95RpVZTdqYDMspquUBfGjZ
-tK5SfkfOQpJEiHfHKTy8GHeadmoohgrGwm8WU6/aMqQoQ/zObG43gWSwfBQOdrur
-6Uns8DNQe8GkyW3hqXld/NfRrFdd3QLy+nvnl5TjC4wvrJ01Y7fRpq9cGAUGrdm7
-t8bY402znmPlZqEnsYu0+d6SLHZb5SFKcV75Rmi7pPIIEoTcFjn7I+rCU1fvEufh
-Jziz2+jIbhHvbLYZRkOnhJCCMmEeH++R0C4g2EqJseg2zPyamAxA/UIVz8Tiorqa
-EqH3zXDY9c7zy9AWUMXCOQLsR73NfDZ1nwwk1ArLtIfp8YAp6weL15N1+SnHfFRv
-sskmOmKQx+H8Q+0ynsQdF9dA5hVX5FCW2KVJROlCaIm6js2jgeWFkkWhDU2P2jQh
-ERS3ONyso/GrnRIReGRKR2UuDEjkX+mi/obO9Tg83dppJ7EKyKLkCyXqanKSRf7j
-5LrrpVOGnvBhS1fDNccVHV6K7aIlVVDuh3ZLcj2RXpYtFR/a5o8QiH2VwxJSDMXv
-R9N15ThZiM5yPm292U5nTBrF3Upaz6x6L4D0ZB7q8W2dwOi4Z4GP5W+O7zeYfE36
-IUoO3QHWjsbKWEqaK7FXu/Vya3dN4gyFHwYifI7tjcqUGcZ5edTdGqgf02e1eMJ1
-BBAWCgAdFiEE6T/bgYSuJBw0XOI1wh/5/ch6qW8FAltzJU0ACgkQwh/5/ch6qW9j
-vQD9HDqkj/+0Ty6Tn2fXJOQJq4xrXKQPB9mJQKmjraTpq9YA/jjrpeszBTsB7sUY
-ne0Pa6NcLAZm5/jYCgzHdHnyzl8DwsBzBBABCgAdFiEE9pr8A1k/XYZrdI2Qi96q
-WVJ4SYMFAltzJVAACgkQi96qWVJ4SYNKrQf/WAbRWULsbUNHbaouDXdBAAa7iBqA
-5i1/OrFz2SB3mXRGBquSjAwdpuve579Ol/Jr1AI7rw/gXRwP1DdAEj0p0ludPbbo
-VAYehOgKRS2j1Ne5EhNgeVwIIRvZe0KpMzmcix/aRlyNcmlNb7X/TCfn5yS7Srdo
-FbIwOij3wZfMiAPn4L3TGmDfvZEvGaweEI+XSKjA9m2GcYUki4r+Wr7faF+eErVf
-rq66T/3VMzsP6gv9q4Su6coN7TDfADKkF20eZIgCPqd4OnSr1oeafYs1C6OUXTbL
-wF1robFb7WTTNGMlOqFCs46tT/FszhkzrvHnYU9AHV5MK/aZ04MK5KnmUMLA8wQQ
+leea2j84TajpniwflYg+V0cw+z75mSq3FV0lwsBzBBIBCAAdFiEEf+2ENMXq98Ix
+/kUMI0UbEHqgOUEFAmTPgJEACgkQI0UbEHqgOUF0oggAwnLQ+yhhgaoPsYXGE1mr
+AtqW+ujyQYTBlQVxTbu28P8WMjc4qUHIE4zOGpcsuNa33mMnZtDnh8TwNYCJwoUk
+J7OTO9Tf8H7XmtfsgwZRcysxkb8Fi9Wr0/G7UfbqGhlV2sHSGdEj+6TczMLyzc1h
+3W5a/UmRLrx2sBMbQDR1yv0cdFhsUmkbJFxFFC+L4mKQHu61IOJOvHVUecBjjaaj
+B35mAcWqjaCAMOQcnnVDBs3Hu4B+5bf/7nEWWuGjGirFgoq3OQceqQ2AFF4rP2XB
+WoiSPKzD71SPBK54HSPnQwnGx6Edu1PD0pIbCJ44qm+uaZHdLzbqE5doQlxfvRgn
+j8LBcwQTAQgAHRYhBOdZ8n2KA3v4ZfcQ7cggKvEFUnAEBQJcqeavAAoJEMggKvEF
+UnAEPGMP/jjNAutUTJsUMt1MnPkR3xY/yd6iRDHu9J2k8tOZet5WpQ65apTjWwSF
+bXLl6oD1Ij4bpP20RjAH5W4Xpb6i1uECn2Gv9msienBu4Z5++c+IrIhXfIwwdHaY
+cz5j0lqgZIJroiQMOcUxbT0wo0yvXYUWkDBmKytHl+2KCMncOWeQG5kNcZn8uo/F
+Zsqlvsf04LAuqVFaNVy986G+Ac0tGKTOv6UyUoTpLuREs2xAOze70TX1G/CDnSDw
+XZcyf7opm5YThHyDsA2O0y9z48cQFKjeP2T/gv0Iol0gXpRrGpJ4Zsb7k+k32/BI
+VXoGkQvoVq6FtgKpWy6fLMDmFsDT7urq8GcHQd+/C+bJnniu6ao3JJHIeOAuKZZz
+YOiRKMwrXLE6YllJG0ghYlXhXzVnZDrdYhj3S7PHXr48jJ5YJWTkalTDyPMk54s1
+uw9wDy5iD0kTN/0x+dEm4ZhAr/tXWor6WdD5UKF2HpVKkYSFF+J4FfmeQUe/Nyp+
+QUQqJOcNVsz3KrEHNAJDrHJj9aUkvl9jmkIa5WO8joX7/hZV8hNBUaTEIprsK6Xj
+rEgftTPhewcub44urOnmK43gQLUFR7vNbk80DywoEBZPsUMyiMgAmxbRgc6LUB1b
+2wrMh9/tov7KxrKeoiwUQu3uisjPFQBk+YU+S7+XEAY9Frl4CTv+wsDzBBABCgAd
+FiEEcPFq2N1m5DfW7AJXaCAPgnigpWYFAluleWoACgkQaCAPgnigpWbw5AwAkhPT
+S+NyvcZXaMqbE8MWiG/O4tsYosSyP9weGWRPU/BiPNJOvJSNyeL78uguBQLk7H9/
+A2UQ4geAAZz5OWG/ZrcmSyIwzbv0Ry1wevxYPwoSpiLBvLDMvOaRRuX8gXdmN8L/
+51E+cCJ/iD6UEko/2RfpWDnPAK/h7KqPtsnpgLWMvXHz87/TSBfbvsxsbFvedz15
+alHSyQw3nyFLYs4ImBcpvYBlAbwoX0/eqdB4+zGNlhZZPt3TLr6ApngkL+2bM2mK
+bySHb7Zmt2IN2mPGQ7R9gZ1nIsPuGxF+fsLX7BqL3GkbpIZsjUVFImRZwcFXRc7a
+CIRd7ZoMIT7R4sY4ofGd3i+c5zXLu7vsCGomuv4K++5tl5gTQq4fDbepbYs2brKu
+cQbN0Kz5kvL3qEr1kEqO6+0JUayS3cPWIWttFpOdrAPha7Bf/dj24Vr32DSMwPkU
+W70lbMDD0zGTsCXerVK387RQECYYyCL9n15c7ww3I5LdWLHEt6rOnFQvAsL/wsFc
+BBABCAAGBQJbjpbdAAoJENsSEG6LfruIfFgQALwQyTpQzrSbx/xC3Wn+0awdFmjd
+xeqzvsHecfM42WEzs+HqJHQtklT02AbO3AmoVpEIcJ2lBbVFYUP4jIKOkC8/Rll3
+M58uhd0pyGA+TiWNwKou2O/cmSWPmgLtFw/wZ6wpAeEk83AHmn62UQy4jqdf0L2u
+qfsFrFVN1TdaPD9HOmUhAgZQHJ7GmcEf5WlS41OBEDYnSiC2foNMQpqkQDqQyzal
+xh7b7m0TnH3YwIhtZQv6If1O7Pmfrygmducmc1qfI2YoiUKY66VxGBGohyUXuvDD
+Lin1xYVhEM/rkaSvKUBc4HZdGVufhExZwR/YHWcFnfROTv+CWfw5L0Q7XpRAaq+a
+/r535N+EhzfSCGt6Ej3Z70CINBOTIn54WMEBhIKC24uL2hYAwJu2Tk5OgULMCyr+
+HIh9jpbDUgJJvewtrK8wsOR/awpEvV+W5JFdrlaZVOrC020ApDagf6oqo1krvKgT
+BHpey6vMtOqmMHUgr4r+FtiQNKzRGQm5+NIiU+WxmwY0e3hCUBa6QTnuQ63VS7/t
+sKPciBIvcvrTCJAqNh6+k2ZdGu7scT1FYDETtdim2ZEoQLqtskwXO5epJSZmE8l1
+euWj44JVP0+jc3YazLLtfju078YSYeWl4H3foIpHds+A0fxyHHfsgFy97xdG1Qre
+ui1GLK2wqiHR0eAPwsBcBBABCAAGBQJbeqIjAAoJECzMPB67dMo7WcAH/jd9QmuL
+l6Q5M8JwrJPvK3/Vjt2EgycfS3Giery1Mgs0uUGaQO0YrtHHC2RAifh4XcwVjmHI
+nXe61l8m0IMQ7EDwmQR97hZCtjQpu2vlV9CyaSMqXizt553e66qPhwXCGtIm6vQc
+STCxdJ1ji3oiStJY+K9ASUdqVy9GwaRB9Bm8uXKMduC0S3hb1GtZ010unrRphH0B
+9Uzgte1UQfkgXQ9Afbzpqnk//892eHOEM5/VDbR/Bh0hJyxuY43cLp9i+rIEh1Ax
+AqyzQXz43Q4dGMgF1oIV0X3IMpwK8uLKorSuESWaghLnNqvf83iZRFV1CbO6Ql8V
+GA9kWoLwUhuVeA7CdQQQFggAHRYhBFMiBEFRHK5DvalSBn+Dwv2ENls4BQJbenmi
+AAoJEH+Dwv2ENls4GzIA/3OnVVGjBwlL0MqjeMynWrznsbfqmcuttMTeW3f0UPea
+AQDi8WQRYPXfQF6UG38OoXiA4Bi+PHb+9tcBW10m6PxiCMLBcwQQAQoAHRYhBPBy
+A/4j1FtM82DmFY3QoM2ZYq5lBQJbeZx9AAoJEI3QoM2ZYq5lqt0P/1NseY0xNgaI
+i7CtzvcCn/MP3ZdoIDHIl1MbzDtwiRJAZjzcekLfm+uSHAs/h3tvRDT1Rahzqy3B
+1YL8XArs0SJj03E+vWUcQoiKSydq+wU1mQlnQA4nhCNvVNh2g55mv3jByvQwuLKY
+XtFk+/qI7oWUfygrp5+jVdrsPpou3DkYy4f4udHWoeUQGTl7VeprQvs78OYAlyga
+eOZL8M6Lqy6HErB7iWZFsUFMaCIbikD3RlMydGXvdRejlNAMWAt2hxy81yZ+CehR
+zPVub8X+pOZ4BKfk1CD5UmU2dSe2KFv7dF8y4tk1IZzPEReVc6u0YCQwWvBfaCP7
+z/5wup1LL5yqyVQs2QEtFj52ne7cFy01lAgq2KlPuIXwIvVXk2fzwlszv9XkCuQg
+Rc9cBL8HxcxasrlJsWGJioPI5vxNBlqrEO6nAltBc/W+tZZVIhWfqYX0xLdgo/p5
+RhGOV5o5+Ds0eDy2k3Ouihf5Lp8oeIz/m8d4qb+vVI0HrX84HnSaVoTLxLjyNO8Y
+fNxNWcQDXtHQfQVhLbnSQ+DhTVs4FQFjb74DWrHhbOPpxNmB9KjRfh732RwCQ4ff
++T9KU70D+QdXYpHfR5ySpcfVgNPoSrEgC4v5gAHm0+z4IIFar1BFCwWRDWxoDUQ5
+J893je/xHK/a/Tw1ZhRCT8alawa3CQ/Lwl0EEBECAB0WIQTmdkh6+ZNe5/tC3fmV
+m8AxX82AYgUCW3lWagAKCRCVm8AxX82AYi+KAKDTrBX2dwcqILz7G9x8QJV9I5Mt
+1gCfYnZsMjHeiV23OKPcQgCCAx9YOAbCwXMEEAEIAB0WIQRUBuzoNmXanSAdNXIL
+rwyce2rp8gUCW3hUNwAKCRALrwyce2rp8lBFD/46NMEps6R19BTyx7EYdvTXVZ62
+REgyntubYWJPHEWAQ3ShAjd+8/nO989RJOA+7HQO4ELCf0aXYTvr0tAgXm9clN7T
+0mdm7nFFYcC7kJI1Q/63LHftq4LWIeoAxjbBu6rqhKf7U1ef+LwkK68qcL5/A28+
+vY4A7ed81st86s2nGH/03KHasu9SCsABHTWKPpmPYzqH6H9GScRAfvwUngwdma8J
+ARd791Tfff3D0835s+V04+TEFaE0RiFKZcsqs6hknnGCEkHCp72tfb/zvrTaMf3Y
+A2gJN8jPOXttevTzSE54kkDKKF1cqk0nI50YRd5vk7Rh8qMOtFCKhUf/Qq4kajMl
+0MaqPOxZeLxpYhbCoq663rXSGll5ZVTXjvd3Q+bzzqtyNI1EYif74pzZStcnBi5d
+/PD3XrnsmLJ45QtTej9RK9QAzXoyrWR1ngD0B4o3CDIx+Dv2cmywe6js5+lHYCN+
+iXY1UFxlcaBGtE7ZdQzWvC5uFo/84KuZ9/oe6s3INFEf5wsjBszdyZjWRkhxkbZY
+7e+bAsXNfxZmdaNVSNrxSzY3zPs5yhVzst8oaCzoPMX15U5tUiIvVqiXvTtXekOY
+1kU1jUbZyl8wAjGUTLmjCPksNuAGlK3UXp4NIeyPwc8OWAuqcPzvN6+Hug2mUp9a
+gXdgZtudhrhYPK89/8JGBBARAgAGBQJbc15SAAoJEKlgsQEWNPhCpU8AnA+txEzO
+n0SFF7jWjg1Huvrxxug1AJ9HpRFhUmbZXD6NB/ZziG8i3D7kzsLA8wQQAQgAHRYh
+BJSlyaA8L+XKOwldjh/fcjz0YraxBQJbcyqVAAoJEB/fcjz0YraxJxwL/1kBFh2w
+WiwLWbTf/q2a+81JLrxPTdiT8Y7E82kBq7bA7Y+Dsa08Mf3RdHnytiHnHDirWhi6
+Mf4czen/rI7CTpzp2OOC1Zy0QFtwb15LtsoBwoVyu9fZ+EQMiyj1Po6Hb5kiSIcZ
+Q70TS1494WJBjKsf8U0WVXfjqgd4pIqxHYTyEcfnQVabyxFgajG4ckbNWlvTWY00
+m/Rd/y87jaPBb3wN+05mCggPrr5pHi5z/ugzsi5QaRT5moI8r+HUxwmkvUO/BygU
+0M6J5BknOCsCvHnpE4ajx1bSZrQTJgWQ7+VQ3wEEghVWzTqU/bdV/0cF0aTyvZPH
+lAz0cogwi3kuVryV7xL1jZeCpFHCSlTx9pAy+Bit/UWKl3VTWsjJFPTWal2ifdUZ
+cw3W8wyBL8it5FtKp1UDplc4exFOC/SmuTNlUYN+ws8ZoVMvYcx1d614Rxt/V7cO
++DDxoXdQfK5aXLZh0VCfHkaM7RGP5v1/gDNX0Wk9JKPEEPwRM00HSSuoQ8LA8wQQ
 AQgAHRYhBFuAxXVCmPDLVdjtarzvfilLCS4oBQJbcyp2AAoJELzvfilLCS4oHU4M
 AIcUayob5N0m8Ri5veRidW2IitgeddNH8vhYzCuPK15nBZ1pJJKzzIa+PFujRJwF
 U7w53mSyIgdie5ZObSRlCzlc30d+KzhidLKJ4vikWPRLz02gd63zYR46mT71g6sw
@@ -251,168 +216,224 @@ OHSFBO6rVBwC7vH5tgivAxUo9C4k/QnunWEbjXGMfGEjfnYnBQ7iGfI2GAE8IzdP
 I+feipsSxke0wGPkHfmlbadAFJufHuQ5YmhgIIsS/kdh17jBdDU6G7QzPnEeTcTR
 hNBKfuXk3dwDyaMfRHqn1qgZPNrfcQevqtmivXWzE09l7SZOlceRUDOgfYt2L5xR
 ih6i7a+b6M+QDdZWOw2+1+W6H3gwnwRpSsndW66OFsxqILgHlt7iWBubWVDpUxuo
-isLA8wQQAQgAHRYhBJSlyaA8L+XKOwldjh/fcjz0YraxBQJbcyqVAAoJEB/fcjz0
-YraxJxwL/1kBFh2wWiwLWbTf/q2a+81JLrxPTdiT8Y7E82kBq7bA7Y+Dsa08Mf3R
-dHnytiHnHDirWhi6Mf4czen/rI7CTpzp2OOC1Zy0QFtwb15LtsoBwoVyu9fZ+EQM
-iyj1Po6Hb5kiSIcZQ70TS1494WJBjKsf8U0WVXfjqgd4pIqxHYTyEcfnQVabyxFg
-ajG4ckbNWlvTWY00m/Rd/y87jaPBb3wN+05mCggPrr5pHi5z/ugzsi5QaRT5moI8
-r+HUxwmkvUO/BygU0M6J5BknOCsCvHnpE4ajx1bSZrQTJgWQ7+VQ3wEEghVWzTqU
-/bdV/0cF0aTyvZPHlAz0cogwi3kuVryV7xL1jZeCpFHCSlTx9pAy+Bit/UWKl3VT
-WsjJFPTWal2ifdUZcw3W8wyBL8it5FtKp1UDplc4exFOC/SmuTNlUYN+ws8ZoVMv
-Ycx1d614Rxt/V7cO+DDxoXdQfK5aXLZh0VCfHkaM7RGP5v1/gDNX0Wk9JKPEEPwR
-M00HSSuoQ8JGBBARAgAGBQJbc15SAAoJEKlgsQEWNPhCpU8AnA+txEzOn0SFF7jW
-jg1Huvrxxug1AJ9HpRFhUmbZXD6NB/ZziG8i3D7kzsLBcwQQAQgAHRYhBFQG7Og2
-ZdqdIB01cguvDJx7aunyBQJbeFQ3AAoJEAuvDJx7aunyUEUP/jo0wSmzpHX0FPLH
-sRh29NdVnrZESDKe25thYk8cRYBDdKECN37z+c73z1Ek4D7sdA7gQsJ/RpdhO+vS
-0CBeb1yU3tPSZ2bucUVhwLuQkjVD/rcsd+2rgtYh6gDGNsG7quqEp/tTV5/4vCQr
-rypwvn8Dbz69jgDt53zWy3zqzacYf/Tcodqy71IKwAEdNYo+mY9jOofof0ZJxEB+
-/BSeDB2ZrwkBF3v3VN99/cPTzfmz5XTj5MQVoTRGIUplyyqzqGSecYISQcKnva19
-v/O+tNox/dgDaAk3yM85e2169PNITniSQMooXVyqTScjnRhF3m+TtGHyow60UIqF
-R/9CriRqMyXQxqo87Fl4vGliFsKirrretdIaWXllVNeO93dD5vPOq3I0jURiJ/vi
-nNlK1ycGLl388PdeueyYsnjlC1N6P1Er1ADNejKtZHWeAPQHijcIMjH4O/ZybLB7
-qOzn6UdgI36JdjVQXGVxoEa0Ttl1DNa8Lm4Wj/zgq5n3+h7qzcg0UR/nCyMGzN3J
-mNZGSHGRtljt75sCxc1/FmZ1o1VI2vFLNjfM+znKFXOy3yhoLOg8xfXlTm1SIi9W
-qJe9O1d6Q5jWRTWNRtnKXzACMZRMuaMI+Sw24AaUrdReng0h7I/Bzw5YC6pw/O83
-r4e6DaZSn1qBd2Bm252GuFg8rz3/wl0EEBECAB0WIQTmdkh6+ZNe5/tC3fmVm8Ax
-X82AYgUCW3lWagAKCRCVm8AxX82AYi+KAKDTrBX2dwcqILz7G9x8QJV9I5Mt1gCf
-YnZsMjHeiV23OKPcQgCCAx9YOAbCwXMEEAEKAB0WIQTwcgP+I9RbTPNg5hWN0KDN
-mWKuZQUCW3mcfQAKCRCN0KDNmWKuZardD/9TbHmNMTYGiIuwrc73Ap/zD92XaCAx
-yJdTG8w7cIkSQGY83HpC35vrkhwLP4d7b0Q09UWoc6stwdWC/FwK7NEiY9NxPr1l
-HEKIiksnavsFNZkJZ0AOJ4Qjb1TYdoOeZr94wcr0MLiymF7RZPv6iO6FlH8oK6ef
-o1Xa7D6aLtw5GMuH+LnR1qHlEBk5e1Xqa0L7O/DmAJcoGnjmS/DOi6suhxKwe4lm
-RbFBTGgiG4pA90ZTMnRl73UXo5TQDFgLdoccvNcmfgnoUcz1bm/F/qTmeASn5NQg
-+VJlNnUntihb+3RfMuLZNSGczxEXlXOrtGAkMFrwX2gj+8/+cLqdSy+cqslULNkB
-LRY+dp3u3BctNZQIKtipT7iF8CL1V5Nn88JbM7/V5ArkIEXPXAS/B8XMWrK5SbFh
-iYqDyOb8TQZaqxDupwJbQXP1vrWWVSIVn6mF9MS3YKP6eUYRjleaOfg7NHg8tpNz
-rooX+S6fKHiM/5vHeKm/r1SNB61/OB50mlaEy8S48jTvGHzcTVnEA17R0H0FYS25
-0kPg4U1bOBUBY2++A1qx4Wzj6cTZgfSo0X4e99kcAkOH3/k/SlO9A/kHV2KR30ec
-kqXH1YDT6EqxIAuL+YAB5tPs+CCBWq9QRQsFkQ1saA1EOSfPd43v8Ryv2v08NWYU
-Qk/GpWsGtwkPy8J1BBAWCAAdFiEEUyIEQVEcrkO9qVIGf4PC/YQ2WzgFAlt6eaIA
-CgkQf4PC/YQ2WzgbMgD/c6dVUaMHCUvQyqN4zKdavOext+qZy620xN5bd/RQ95oB
-AOLxZBFg9d9AXpQbfw6heIDgGL48dv721wFbXSbo/GIIwsBcBBABCAAGBQJbeqIj
-AAoJECzMPB67dMo7WcAH/jd9QmuLl6Q5M8JwrJPvK3/Vjt2EgycfS3Giery1Mgs0
-uUGaQO0YrtHHC2RAifh4XcwVjmHInXe61l8m0IMQ7EDwmQR97hZCtjQpu2vlV9Cy
-aSMqXizt553e66qPhwXCGtIm6vQcSTCxdJ1ji3oiStJY+K9ASUdqVy9GwaRB9Bm8
-uXKMduC0S3hb1GtZ010unrRphH0B9Uzgte1UQfkgXQ9Afbzpqnk//892eHOEM5/V
-DbR/Bh0hJyxuY43cLp9i+rIEh1AxAqyzQXz43Q4dGMgF1oIV0X3IMpwK8uLKorSu
-ESWaghLnNqvf83iZRFV1CbO6Ql8VGA9kWoLwUhuVeA7CwVwEEAEIAAYFAluOlt0A
-CgkQ2xIQbot+u4h8WBAAvBDJOlDOtJvH/ELdaf7RrB0WaN3F6rO+wd5x8zjZYTOz
-4eokdC2SVPTYBs7cCahWkQhwnaUFtUVhQ/iMgo6QLz9GWXczny6F3SnIYD5OJY3A
-qi7Y79yZJY+aAu0XD/BnrCkB4STzcAeafrZRDLiOp1/Qva6p+wWsVU3VN1o8P0c6
-ZSECBlAcnsaZwR/laVLjU4EQNidKILZ+g0xCmqRAOpDLNqXGHtvubROcfdjAiG1l
-C/oh/U7s+Z+vKCZ25yZzWp8jZiiJQpjrpXEYEaiHJRe68MMuKfXFhWEQz+uRpK8p
-QFzgdl0ZW5+ETFnBH9gdZwWd9E5O/4JZ/DkvRDtelEBqr5r+vnfk34SHN9IIa3oS
-PdnvQIg0E5MifnhYwQGEgoLbi4vaFgDAm7ZOTk6BQswLKv4ciH2OlsNSAkm97C2s
-rzCw5H9rCkS9X5bkkV2uVplU6sLTbQCkNqB/qiqjWSu8qBMEel7Lq8y06qYwdSCv
-iv4W2JA0rNEZCbn40iJT5bGbBjR7eEJQFrpBOe5DrdVLv+2wo9yIEi9y+tMIkCo2
-Hr6TZl0a7uxxPUVgMRO12KbZkShAuq2yTBc7l6klJmYTyXV65aPjglU/T6NzdhrM
-su1+O7TvxhJh5aXgfd+gikd2z4DR/HIcd+yAXL3vF0bVCt66LUYsrbCqIdHR4A/C
-wPMEEAEKAB0WIQRw8WrY3WbkN9bsAldoIA+CeKClZgUCW6V5agAKCRBoIA+CeKCl
-ZvDkDACSE9NL43K9xldoypsTwxaIb87i2xiixLI/3B4ZZE9T8GI80k68lI3J4vvy
-6C4FAuTsf38DZRDiB4ABnPk5Yb9mtyZLIjDNu/RHLXB6/Fg/ChKmIsG8sMy85pFG
-5fyBd2Y3wv/nUT5wIn+IPpQSSj/ZF+lYOc8Ar+Hsqo+2yemAtYy9cfPzv9NIF9u+
-zGxsW953PXlqUdLJDDefIUtizgiYFym9gGUBvChfT96p0Hj7MY2WFlk+3dMuvoCm
-eCQv7ZszaYpvJIdvtma3Yg3aY8ZDtH2BnWciw+4bEX5+wtfsGovcaRukhmyNRUUi
-ZFnBwVdFztoIhF3tmgwhPtHixjih8Z3eL5znNcu7u+wIaia6/gr77m2XmBNCrh8N
-t6ltizZusq5xBs3QrPmS8veoSvWQSo7r7QlRrJLdw9Yha20Wk52sA+FrsF/92Pbh
-WvfYNIzA+RRbvSVswMPTMZOwJd6tUrfztFAQJhjIIv2fXlzvDDcjkt1YscS3qs6c
-VC8Cwv/CwXMEEwEIAB0WIQTnWfJ9igN7+GX3EO3IICrxBVJwBAUCXKnmrwAKCRDI
-ICrxBVJwBDxjD/44zQLrVEybFDLdTJz5Ed8WP8neokQx7vSdpPLTmXreVqUOuWqU
-41sEhW1y5eqA9SI+G6T9tEYwB+VuF6W+otbhAp9hr/ZrInpwbuGefvnPiKyIV3yM
-MHR2mHM+Y9JaoGSCa6IkDDnFMW09MKNMr12FFpAwZisrR5ftigjJ3DlnkBuZDXGZ
-/LqPxWbKpb7H9OCwLqlRWjVcvfOhvgHNLRikzr+lMlKE6S7kRLNsQDs3u9E19Rvw
-g50g8F2XMn+6KZuWE4R8g7ANjtMvc+PHEBSo3j9k/4L9CKJdIF6UaxqSeGbG+5Pp
-N9vwSFV6BpEL6FauhbYCqVsunyzA5hbA0+7q6vBnB0HfvwvmyZ54rumqNySRyHjg
-LimWc2DokSjMK1yxOmJZSRtIIWJV4V81Z2Q63WIY90uzx16+PIyeWCVk5GpUw8jz
-JOeLNbsPcA8uYg9JEzf9MfnRJuGYQK/7V1qK+lnQ+VChdh6VSpGEhRfieBX5nkFH
-vzcqfkFEKiTnDVbM9yqxBzQCQ6xyY/WlJL5fY5pCGuVjvI6F+/4WVfITQVGkxCKa
-7Cul46xIH7Uz4XsHLm+OLqzp5iuN4EC1BUe7zW5PNA8sKBAWT7FDMojIAJsW0YHO
-i1AdW9sKzIff7aL+ysaynqIsFELt7orIzxUAZPmFPku/lxAGPRa5eAk7/s7BTQRX
-zWQ+ARAAvXt5B0fcIQNSAGDkROWL37339U/rpsQ8IoLMYPp/r85RdstUFezTcbMM
-nMGvpvzKxpYxHtzRZcIoV/tTn7dAjiz1MGVyycfGrn3fjecmHZawTlxzvlYeV8rC
-x7vvNPe26N2zYyNIGPbfbiyTD1EdKbiaRzt6xs3wYJBEAziBSPASjvpJe9XC5Qph
-pK00tbwMotocHE01yWTe/+TthWZWjTu5/+7y8gZ5S+qX+XZcGVZaHGSBob5SdHA1
-ykyHSK5WSslifTgVC05XbQRERUhMbW1YcxDgrOep2B29sRacWSRse+PZzqsbKVIl
-KFBtr2kTUsBs2EXqFsSKnaerq4qxBPQDPb9la4/zcYfjIMdskIOhPyV/bFxrDP7J
-93ar1AXxB4g5ssI23T10k1x5NuNTeHcEQC8PqAZ2rMcM89+Z/RRWOLXnvo8SaF4y
-7bn/NuHWa+qexXZgdmwZdWlsEHU8fOsozUMfc9+oVMVDl6Wu+S7GGBGtIpqJYG5o
-YPsNYyIQhXZAqZi63FhLQUnnuUVFoqjQCJW+0MtWHR52OQu+SC5/GzqgA+cu3mS6
-j+dr+2BYmNjbQS7rd1C0huk9g+sAubb44wrx4Cfe06DsexQxdBLmECpG8f/xoxWh
-buqMyV2C2GTkRdOmXKWQ0II/P11OpP+DxEGYrF6WdF+CPcLHLh0AEQEAAcLBfAQY
-AQgAJgIbDBYhBMomLGyD3k0vsoozKjpqTbg56qbXBQJfIJlJBQkJSYCLAAoJEDpq
-Tbg56qbXQ34P/iYT+MR25JPpsKmcKM2wxTKeMdMJAnuD68MFeWQ1wZ/rjbBu1oyp
-4/SXYywqmbCt+UB2zocdr7sN/Zmynt/K3y39G6/1r0SZIN/d5EQ1ukS65VJ6Mo79
-7AAOTleUpIZUQs8BTyC1xsqvMg4yqvcYF7gIfzeyGi42JkH/59O2/rVMmvqPJ4A4
-LsAZd4xUkMnIi1kZX9zwLMfbYJGBTLSuG+Sl9K3BukWRFTzST/V28T2TeDZAS25i
-INMq2/SEnaDxY+xgbaJxZO1T7rJNBmLSJ8yqiwaMwm497+Bi00nRMPmyl8pnu8aa
-Qk1wTbzsgDNmkwbmi8huh2OibOXYZnu5j0UMLpxkk5JiJY6DIJv/q9jlBy944ldy
-TvjQITn4Ew3ifbi07dz6n4iLApdiKmhnvNGxLGTcj/Pet4j/ZAHJXnjcOVrBmQ4E
-duMipxRcVwFFAbJzsxmzm0nGhQofdvGRzjOQ8u4RT6rxIVT008bxGJ+E1ynOTXav
-3mOzIW8OmsBJEdJyHxz1uXns4hmSzB1bpXTzuB5/YswuhIF/Ou5T6HTOYInc8+wP
-a5uIhlKHjQfbW3039fPlIleGUPKxCp/jK9/9hX7BD4gbsUtq5xquaPyZ/kL1kysX
-1D7yAZlDJARnS4Xj7pHIENRMZ8igj3hWp4jnqRgKUdaYxFvzcCS8ghlFwsF8BBgB
-CAAmAhsMFiEEyiYsbIPeTS+yijMqOmpNuDnqptcFAl1TtlEFCQdnhZMACgkQOmpN
-uDnqptd7jhAApYjTclVSnmRawq+QHYDPAlw6A9W+Mkrs4s/V4/JaJR7cR0Lu4KL1
-5AHjBzSW66zQySwg9i3ZZ3TBRz2KZLNJ2Wnaied6EWH6HEqyYsVOyIAxzlPnqQGg
-n5818WsJi4hm/vx55vgVB9gXFeuY4vhzekknfGaAvwH64V26NHl9KNvRgGiz4m2U
-lZQXHaTrxDy6iK9yS7+EtSDGIs+UAOsz01OL66wdbxxfZWsbKgVfDokgg1mtnAcd
-mvO4fk/K48sOiSPoUxzmcJaS+NDFhYSpHu/4g40O/wIR2PbbizgH9A0tcU/yb9Ie
-eA6+ioFvGgYkPH1ur8IePuk59RBy6vxYSjPSTK7eq2+LWdGZ8xSnAJZH/Dd+Zwy9
-wxL4CwLpKos5Jwf6hxzJzJ/nrME8IItwAClP0Tl0w4y+zyaKpgFcfcDiIpPH0zJ7
-Sbuddn+fVjxDoYH7uHxBlQ4mV4hVZkvY7dI176n1SNlRhJe5GWTn3i/8Vr/TIa6g
-jl6+no5kISLaFTmo/ku3iXEjeLqBhkgYe8NWI/snZd6XGDcTqbOP/bAWmYE/NlWX
-5Dj3DwL2Md81r1ypAQopiv1YBy5XK4h5c++Kk5/65DkPZrkaJ0kvfUsXcoMTuTCx
-b1YA6Zu8L/sKLYdBoHUC/9T9aVmuVXPs+pnXyTNNolsDYLeM1KpNXj3CwXwEGAEI
-ACYCGwwWIQTKJixsg95NL7KKMyo6ak24Oeqm1wUCW2DJJQUJBXSYZwAKCRA6ak24
-Oeqm1/usD/9qul/9GMBFJ00b8qw5BNvHuTZIjWHitekGy7zVcaaOYZvJ4woNjY/b
-L2MBqflp6qiXMrKwQN//vmT0Bn3bJKigY7zRvibKuXV/o0fM0GyWNev85BXGQ1Sz
-/IxLHsq/xVIk6MVEs45k6kfIPGazVaQ8XRMEOsxhFIqcaKPCAUJZVWQoJeIM1apV
-dwL1/lYkdWSnpSAnWdejXh4GOsRUS3OjAQE7JF1fNOLsJdB2Hk6euVlIyToPs0Md
-2/XyyEdNr9+I0i5AHRXGdmL3q1xCbPJMV1M1C7tPqqpczH76A39rEzbWvdVA57Wp
-ySlcTXpY1ki0Ub9Qu7t7z8x74kwmOoNxNXX53o69lo2arLm6vMLHWfMdLmM5C/br
-LhqiHopMTaWd55hEDUbe3pgUaMjKJkcrxxpHv9pEhpdW/Bs4PPQgCzjfWRwbUjZz
-CzRUDhRdKKkXfqvLRZTb52wUBgfrdplUsKSkJKHYCQHAoq1hrNVyPDuWv8CnGHFC
-Dyhzd2nZvEG6CbKP9SZDjec/8hoRejrERuvcMymT8GnjXGWbMD9JiQSyZRbB3wki
-EJEz+khkuYg2K4XC301ywgDmQ2FaDC1G0Djrmcto+kHetV8qNtLv3OjovASttB5S
-pO3OC8e9r+DLHvOa0InWmUdA/L/ejIInrVudhMiS43rcpLte1+UJjcLBZQQYAQgA
-DwUCV81kPgIbDAUJA8JnAAAKCRA6ak24Oeqm14MbD/4+nSzki0rLp5SlotmAiMYR
-Q529WyP/a6LZUaXUHNkt7ihzAdoCi4iECxSYntlyIZJbU5JMZDlDDyNDUQMILaTh
-FnYB9gnhLxHaiDiOi/2zTw7qqdMol8Filrt9Kvfxvijt55y7TbWivHSkrKJ63dlS
-zku7D1Tew388x6UcCSNkyRHfkOcvdOX3fJIz9gH1VHzlssFwXdN/ddliLwxfIPgH
-UU1/4FYBHLGFCGz9LgPDhy58L0MqPPJRfA98swQ1S1Zxu7Fp7qR19JXTkRBxlKqY
-X50AUwFKQzP7dpcgf3l3r+GteGr95ipaAQmvFkHnnJx1Rx+fbcvv+vXSBUxGrtmR
-ITFKAPkMl1428WtY44lrY0OJn8wJ8JKgobv077QhB8x64rUs+yPcAcZMY5aHo6AA
-spV+A4eTBWH8IU/n5otBNNRTuVK2E93+PC+rBdQVgW8DB2DxWNBJfSRPrZkjlRmu
-p6MQlcOm5jIFA3kFlSCrCjLehPLQq1qjdN31JQMSkVopu3VhZaZgcuuxxW0jrKIo
-swi6wdRcxr50HuIoYrUGNTZnY8Tg7DsNeqdoQnBI0SWCSF+2dPeUirSoVMNKR4S9
-9mDd8y0il0iJieCtmj6/Grs05hDpEse0Mte2BgQJmtBfo5uX7eN9cVVvwS2G2XWa
-GkACL5WEL6xeHQ0w6rinEcLBXAQQAQgABgUCV81qOQAKCRB/6j2mFpx31g4tD/0X
-l9kAxrL3V7unzRQT7rLFfXYiYB830yo7zImoHXjHy/qWr9KmTTgyqwM/BGDuMzqm
-NMJ+eYmPuTZlqhucp4X6PHqZ0XSSumX9mRH1dQAQ6LSscgIVnx+tbJheW4wur1wF
-b93Rwj/1jScEmHpuWzte3tKLqHxo6dky1ahYMp4XxmCWXcvWstrPahS8dKNV5Ct9
-4Zqel15kZ7jKDN1bjO0WkmiERUhVXYNfnnnW5hfMIcZRMYgTSootauMohHXM3y3a
-0ZSR4QvJhZ7/5WeDsQNDgGQUWNCjHKi/9f+o93d7hSX1KD77Y6rjcjHn4USxR5DU
-CCA/EYZcPEH0F8y8Nh9Y+NIlNkV7le+bmQvEEMFLQB3RanU7RlxXpXmUZZ9wTSCn
-BaUeIo96OqiHeQlv/gcEw8EZcANfn8SvjRHCB/iMuJ9j/OwrHBCUD5jV8YnKjoZO
-66TPuVN7V6+EILtkc9mLkxp6GjscyM0NI0EDOeHWTpGc9OXtMwYYzVgID61XWSKl
-8ti1YWz7yj3ztWMrhiX9wPlmyp1yNU3aj0m7WHAUEIrFTekUXHPW8cLDx4qIfUwu
-8JZaV97Qdb+gyZu1A+JHBrkxK86pgsmvlQY0Pf/r1oj9+YsRL26bafATvf6hC8EP
-T198z0j7xnyqXbwcT51ETuX5e1gbw88aP0McXGuf8sLBXAQQAQgABgUCV81xGAAK
-CRD/JVEZb7j9gNwtEACvLtyNdHULqwzb8xlPyd8iVuVAJc0pFJpvoVkZPOgMzzuj
-08+oPSJ/QFgUIQH6uu+26iw22lSO+1a+evlF/IhlN8Q9N5oQc5QkBBWgLMZ3Cxox
-dbP/bHe6zsNuvFiOt2uemJBzwgaDsZ1cJxQM68mv1FEOhYaoeFfoGs9mEAoEWVef
-Eqot2MMlute3jUm9rv88EHtypFAffsnb3tHEDubKqV9Qy1rBmKmsJtN5MwlpAErk
-2hTDHYmQE8qZteBmWNRD3E2erCPGX40NSUIFjUjmS5xk/KuUIftw17yV41W2mhB3
-fkgARGWZ4YKDGz1QxF+lofdaVVkKDB3PsmaQz9vetVUjHRwmuwwJTjze1adatuX0
-20B8zEfm9fwcLqKyYnmpsTUjc/W9F9piNObD0f315VHKP+JYx7jVYKlwQ8QiLQFG
-MFrNRL8adFl30mHGeNJ2QpHS2JUB4uXIjxdcGZPQxsdu6klZJ/m0mq0qBsE35786
-hNtlucH5TqQpn/KFF16ZBE2Cg9zgeL6QtmBS6y/UJyNMvvq6Osd6r6b4GaeKNZpk
-fCFZUA12mbQrSgKI/+byPp+RvFhLwhh9/3lKcgXFUiFFICTjjY0IR23DAzmLP5dE
-WrewqSt+R/4Z4m8Cg3+Soe6Px1skGtM7tRP/nCB0Q4earqd6fQjWqCwc+fUtkMJ1
-BBAWCgAdFiEE6T/bgYSuJBw0XOI1wh/5/ch6qW8FAltzJU0ACgkQwh/5/ch6qW9j
-vQD9HDqkj/+0Ty6Tn2fXJOQJq4xrXKQPB9mJQKmjraTpq9YA/jjrpeszBTsB7sUY
-ne0Pa6NcLAZm5/jYCgzHdHnyzl8DwnUEEBYIAB0WIQRTIgRBURyuQ72pUgZ/g8L9
-hDZbOAUCW3p5ogAKCRB/g8L9hDZbOBsyAP9zp1VRowcJS9DKo3jMp1q857G36pnL
-rbTE3lt39FD3mgEA4vFkEWD130BelBt/DqF4gOAYvjx2/vbXAVtdJuj8Ygg=
-=igo2
+isLAcwQQAQoAHRYhBPaa/ANZP12Ga3SNkIveqllSeEmDBQJbcyVQAAoJEIveqllS
+eEmDSq0H/1gG0VlC7G1DR22qLg13QQAGu4gagOYtfzqxc9kgd5l0RgarkowMHabr
+3ue/Tpfya9QCO68P4F0cD9Q3QBI9KdJbnT226FQGHoToCkUto9TXuRITYHlcCCEb
+2XtCqTM5nIsf2kZcjXJpTW+1/0wn5+cku0q3aBWyMDoo98GXzIgD5+C90xpg372R
+LxmsHhCPl0iowPZthnGFJIuK/lq+32hfnhK1X66uuk/91TM7D+oL/auErunKDe0w
+3wAypBdtHmSIAj6neDp0q9aHmn2LNQujlF02y8Bda6GxW+1k0zRjJTqhQrOOrU/x
+bM4ZM67x52FPQB1eTCv2mdODCuSp5lDCdQQQFgoAHRYhBOk/24GEriQcNFziNcIf
++f3IeqlvBQJbcyVNAAoJEMIf+f3IeqlvY70A/Rw6pI//tE8uk59n1yTkCauMa1yk
+DwfZiUCpo62k6avWAP4466XrMwU7Ae7FGJ3tD2ujXCwGZuf42AoMx3R58s5fA8LB
+cwQQAQoAHRYhBEKwygAMJaAGyXqg5HiXoCYnRC2UBQJbcyVIAAoJEHiXoCYnRC2U
+GEcP/0HtibEBZduU+7N9HQ78j6VGP3lGlVlN2pgMyymq5QF8aNm0rlJ+R85CkkSI
+d8cpPLwYd5p2aiiGCsbCbxZTr9oypChD/M5sbjeBZLB8FA52u6vpSezwM1B7waTJ
+beGpeV3819GsV13dAvL6e+eXlOMLjC+snTVjt9Gmr1wYBQat2bu3xtjjTbOeY+Vm
+oSexi7T53pIsdlvlIUpxXvlGaLuk8ggShNwWOfsj6sJTV+8S5+EnOLPb6MhuEe9s
+thlGQ6eEkIIyYR4f75HQLiDYSomx6DbM/JqYDED9QhXPxOKiupoSoffNcNj1zvPL
+0BZQxcI5AuxHvc18NnWfDCTUCsu0h+nxgCnrB4vXk3X5Kcd8VG+yySY6YpDH4fxD
+7TKexB0X10DmFVfkUJbYpUlE6UJoibqOzaOB5YWSRaENTY/aNCERFLc43Kyj8aud
+EhF4ZEpHZS4MSORf6aL+hs71ODzd2mknsQrIouQLJepqcpJF/uPkuuulU4ae8GFL
+V8M1xxUdXortoiVVUO6HdktyPZFeli0VH9rmjxCIfZXDElIMxe9H03XlOFmIznI+
+bb3ZTmdMGsXdSlrPrHovgPRkHurxbZ3A6LhngY/lb47vN5h8TfohSg7dAdaOxspY
+SporsVe79XJrd03iDIUfBiJ8ju2NypQZxnl51N0aqB/TZ7V4wsFzBBABCAAdFiEE
+5e16J9/c84GC9eQ3EnD6a31EZocFAlmbV9wACgkQEnD6a31EZofIpQ//fEjizSo3
+Nvz1G1SIc8zoV4dG4nBkvOUCStgxXEj8iTZIKIXjdGpwVcGOCOHTak557u3E3MSb
+p+DHwskyv9ShQIti58k7e4TjNxnyFDNo152nPB0wXVrJytsCWfXXesgpST8+j4qX
+ntdVKlcu94fwOvXf/trPqkikTrneyzTv5gv4q4MkDkwo7piwPitZ6hzpM2kRlUkE
+ZY6tX5tLMtKnYYTFLBf8TuSax3bH3TTcUdLPJEdVhRtqbE6mnwiBJxkPgUTRPmfK
+vmq/061gEolmNmpyjHXQNwM0yVTdpTgX1fsZVBsk7CLvGCdkQ216iA5x1tsmYlWh
+jy3g51wXpZPRL9PuXo4mZW4T+C7UE3KsZpffIU+rWFDynutjUzJ3hXldcH/IgSVE
+biDxKvO1eGaossYOTzZVmxYEZDE6uFfWWrDhR32sMlBhz0aKGNGCVuaVJVgxkCmP
+4rOeR0U3QeaMaX2LFg+TxuOz0V+1otBMj1xDBOkz4yQJJdUQfGgN6/XSLYYivXWO
+Ft6b1CX04yRWFlnz2xhHuyxC31iQpcv1+rVKPFVICEDUwY4d2sBT2Up3NN+5VrBU
+jGf8i+o0Gjg9s2n9Y2eUVnuPlhTHJbTLKwHJRuk4p6tUAieWuC3KO3s7uvL/nqtd
+BJn/GUb0UkgJx6eeSaS2pprLabr/pPPkytPCwFwEEAECAAYFAlmKrZQACgkQNYja
+QTyYl5t/5wf/TgQ5gXkDLmDUrPQnbBNPKbT5d0d9GDSSEyI5CWMlP8H2Iq5UGVm4
+lMbqZXG31oddkX8b6iTiS15S9Wx9Zd3ypz1g2qGtw18s+aj2ZmEVRKR7Hsx6npAR
+4mxq7tsrhjdMOqMEWyFa+x9rwFeB26EZvuQIR/mNBDoTBdfkubK4VHDYcke614wF
+f3xTi8pKcUVXxLex0nWNFgLPqKHLnBF39LB71fieLecedzVjno29VEVMUrHX8eVm
+RyuDbBcyMEbmbeArQBHM+gTOUu6zXiLqNVACVfoqTYJWxTbLFOfLHmiJ4RO+pEow
+FvDM4QjsT6A4qQ8VlMeblmYwJvGg5uV8EMLAXAQQAQgABgUCWX8/eQAKCRDslNGP
+fwWZftO1CACqrTsuj0mTnwPgPXgSpuQc66a+axhuOTBaJgu4FXrNfY9pK9UU7+jW
+qj7R4ghwVY87eOLp8MxgGGXa265viEoTQNsrN0IPN8ytT9KJhJajx6ncqteniU6L
+fIUkLPZYiGL+//mvP3AVn98G59hWMg8phfO5lMhJyLLavbP1fmPHXBM7Z2hbZYF/
+9EUICkXShQEFlj2awUluYDsdT/aJ+XeJrwIkuvZ00sV4V4sg2OrK0t8rBxVAdXcY
+5Y1sXc1meQn2y/De6NB+LHsa9xosgsrPkH7QFYVWYDwEgaJzrfGCGtiTvAwkebLO
+w8yK1Tg5I3nxDBIRkCAd06b7aEnszKQ7wsFcBBABCgAGBQJZfuvbAAoJED/btVCE
+zF2EHfMP/R4UFAt+vgeUBkgg82Sh7lq8YgAxosNLYbUj/cZ+vrr66SqWuR2gTQVQ
+KsSlS0yf0vtz6bwKA5W/DMN2x7psGx4rMWwjivr74i0hxy1wtvS1xNiI0YoUJwuF
+rKsrQ8Gqx00+7fwJBb5r7XXQd9+eimwpxD/Co0uykS+dPw5A6imSyW/EqMPVUJ37
+hl8A6N4AhLjGVcg+iKRUYsxwoo6WeohLNMexx4HIIGwSv5+PV9SgL/dcnEzvpedA
+XfNwoQpgtIZgm5krYELNumWsQchCPX4Z7Koe83Fe1ZMnyxsOSsYCsdShFCB0wsqs
+kXXLxkSEQNUVl+LbDo0mYxErFpQw4dpI4W443lX9lUg+qGmNeE6rYzUbnJgz2DHZ
+nRTCuOo5O3kTptO7BPzfV8BkhaCh+Xwf4k5GTdFJETc9g0ImJQqH2CJ60dRZrjOt
+mqs3y56DPi0S3PaPFwxviC72aihtOaBwX9x3W6/XX0Ocg/tgYuWKlqCrVCXzohkv
+4iej9PdHSbyIusTPjoBdEmZpGFMOm/+fhfoZCqK8aD52jEhA8jnK85QsbS4k7vcp
+ORk4EfKCkJRoztnA0u0gUKzeHw4d4sYDPSklp0/Tks29bhl2c2PiIgugZvlOTPDE
+pqPi20Gk/3Z38f+ssL1QMBl1yMt/ZGRvpXdYzGAcJlJIlg5rPzRIwsBzBBABCAAd
+FiEECqx3W7ZDeo2a96Os/geEEX+84R0FAll+sMoACgkQ/geEEX+84R2UWgf/YFd5
+pkD8W6LpIZ46Og9o5Q2SuXv6idCIJ40fTuVUTohbT5R3tdQF42M3Gva0iwHyri4j
+xdbcxhAuc1sQ21eIEPn3J/C7fIhyby3otR14bfU9Q8jTFN544AL1ZqSDz5ocqmIo
+aDCHPZ5SVwZUHQFLuE2azzsMVyNb42pwqLR1oaB2/0ZZbbBcrFDyHMDqPtYhawRT
+AD/tHHfiMMCDCMYOvaxhsFPdntH6aLQXkXVZ9VUioAOkF41ghEY3OS6t2QDLS0zs
+5AoNElOl9VxW/+FYHyag2aW9Ff/7K4khuZHKU4jfP2UxMRXzUwY2vyQnrkCpQKJN
+2guipSwBtw3S0PY7ocLBcwQQAQgAHRYhBBmUkN3sirW+Ouy8PozbACMAeiXdBQJZ
+d20fAAoJEIzbACMAeiXdrbgP/23e1duo35XoGqtNRGjZVPoIhExzXRWKzg027uJ+
+wyLvWH1wXG6JmL/DzL/8KRzexVxZ+urGiaOC0vAZxyV7pZHWmz4ug6wECdSTAQ8j
+1btASDfsNrMsNUM9JT393FiHy5C3JJy3BoysTqV7lZ5DrfmOwbVRXP8/TS2ETMMh
+mnOU/eF+ATz6ZLw7WRHG58B2/kemIcy5NVXJVWj48v1z/STU5EiR02VuMUME7tZX
+BVl0m19GPiAe7AsjjlM79/MF1gwAcg7xFoSidLbxnyFrEM66kPNsJs4TUkS9NZ+Z
+2gKV4a6hze3pUc6txLNitU2XuxGkWUpGOZ0Firu0qYGwubggld60Y1RUREpofsgc
+/YrqvchaBTvvqyZ7Y5CWEB5b76oEeG2jtT60QNh4ItrCdIo0IBVLD0Z5cjhFgq7X
+IGd0nrjAYSnbfIhNyvN9syzTd+qPmAZeuv5dS1NaNhv0zCSpFWPnRVIBqwRx+I8k
+pK0+jUtsGnyBrd5UWMOq0G2Zs0VUMLd+MfVwb1HfhS4Pu2QUwuMG6KuCQvklKBDl
+8o2vJg15JSAt0wN0auU3DIg3fhJ3V03C3wWUZkD6qNOFfi69yg2qKyeqq6Mo9Qu3
+1eZUMCnILNyo6Pm0fCRB7M0X8rvtJnyBw5ClVeFI+j69iCejCVNsO849mxsAMaTC
+WRXRwsBzBBABCAAdFiEE0vqrYh9hjUJoW0J7Y9cmTAVofX4FAll3am0ACgkQY9cm
+TAVofX7hRQgAqAFRWUOyxAKWJ2WU3uBjhhLpejscTYM/4QzhfttYNXTa1MhnHaKy
+Q3u4Sxvr9LCnZiOBEJjZ+7swOudrSG3Uz8Jr7/vyerf7uLu8eHFW1t+4hQ6Lxb94
+hT5eCPKC4QD0CGzXxoBhZdCt5Zn8MkoUkgnNsy1RZ8dZzuwJhrObjHHl9TGQJ4ji
+PL5UGRFJKp0SIIzqEU8uzp/frRXSTISDYgZmK1Vdk5StzLZY//DMw8mMMSDOlD/a
+sc9WhK1Buo0VhVXPiZlxWouhb7xcpXk1W3yUxOcdhM85GiI/wBFkyNRYUxOiUsGA
+MeQdi8aaLjHbfIVgM9RlWuMG1JqTh9wL1MLAcwQQAQgAHRYhBPIydeS/EK/B32kU
+ptvSzok+LRyHBQJZZABEAAoJENvSzok+LRyHnPEH/0wiVQW8C5+bcR2X84zA1+WM
+0yogGpDk2K4hISGGTzXLEYnwJpUmc03sR38jgy9V/xREmjzHqR7JN/wM7xGbwUBj
+56GqJSrYPTNIp5WM9nJrhaGScA+ESonipYeGPaLPe0Ktr+txS4HHuggAU0L3MgZD
+kioA6ZlZwx4+23eqvI2+j99yVcO/ghooJfDOSjzQL7dG4e+CwQ37Yv+FCZz4l5Lo
+0nNnBIBUncu9Za7rcXBjfOzXzPy0jsVA7V4E4uQINxjeFgkkuvPxXw/bL0+JDhB6
+WQqmw1PYUxR/0LN6iYU27ykW2lidzS7dNV3coh86U1eOQMfbvJhLU5o2pcsNoO/C
+wHMEEAEIAB0WIQTwfYXKoYrPRqNG/QF8f8bqhjO06gUCWP0YZgAKCRB8f8bqhjO0
+6jXfB/0Q68/0kwjGACjI7cuyPTjmEWK/DEQgn7SoSBnjO0BG6BO1jeXNx46EoO5b
+2bbCO6E5OaW6iZPBomRlpJqYJS04lEy4WAadiVT0np1QII6nmMlNmh0ue0Ythj0q
+DNre4kriv/Lfq4HU5YVqcKUWG7SA5mPhjA0CuTuqojNMDC7bukv/slRp40FmyTGL
+MzkyZVayQi6cmcUxQI0HQQznj1/j2J1Fv585pPCv+F5E3H8UGbN88OvuoO+x99k/
+48A8b8cqdWDYfvuLvyAGpbx3/ecpazR/AyUfH1AwvCFBHAYR9Tr7z/s01rrJg2pq
+CjtTCpTLFcl7EHVX2OCGn9uvqhItwsBcBBABCAAGBQJXzZ3QAAoJEFjQ7mSKSLO7
+U9MIALx6GhTzB7m8LyuGDWk7Jq3VJQIiCTXy9YHIvo6lo5WNwpTvRvWww8mFqXOX
+JLt561cp5eJ3fGZV+RBH+YYH5Ju3j1yVzLBf695yWTUAFwFG0dHLoUN138MqmRr6
+YmpXIqZ6rSYLlonXzOO1r+dLJhGAbXqK8E3M7pBveYuG+Ssej0iQhdnXgBlyb2KD
+8oNKXCzAdi2P50SR/9Y334t5Vb60Y+DokJpNd2xJvwIa0I1McYiDH1dyyECAfzN9
+/oXTljP1eIdNFAWdBVLQiog3hVf+iUGllwJwGPbEw/m7EpBMk5BF7ZQmrGtBpszA
+A3uH6jG3L+D/jEi8WE/YbXskZJzCRgQQEQIABgUCV82c2gAKCRDvYpxUCbBuEIhB
+AJ99nM2jdPReVugYxEkSb04jine64wCeNWiJiPbFk/d+1RenJRh7ulhK6TjCwVwE
+EAEIAAYFAlfNcRgACgkQ/yVRGW+4/YDcLRAAry7cjXR1C6sM2/MZT8nfIlblQCXN
+KRSab6FZGTzoDM87o9PPqD0if0BYFCEB+rrvtuosNtpUjvtWvnr5RfyIZTfEPTea
+EHOUJAQVoCzGdwsaMXWz/2x3us7DbrxYjrdrnpiQc8IGg7GdXCcUDOvJr9RRDoWG
+qHhX6BrPZhAKBFlXnxKqLdjDJbrXt41Jva7/PBB7cqRQH37J297RxA7myqlfUMta
+wZiprCbTeTMJaQBK5NoUwx2JkBPKmbXgZljUQ9xNnqwjxl+NDUlCBY1I5kucZPyr
+lCH7cNe8leNVtpoQd35IAERlmeGCgxs9UMRfpaH3WlVZCgwdz7JmkM/b3rVVIx0c
+JrsMCU483tWnWrbl9NtAfMxH5vX8HC6ismJ5qbE1I3P1vRfaYjTmw9H99eVRyj/i
+WMe41WCpcEPEIi0BRjBazUS/GnRZd9JhxnjSdkKR0tiVAeLlyI8XXBmT0MbHbupJ
+WSf5tJqtKgbBN+e/OoTbZbnB+U6kKZ/yhRdemQRNgoPc4Hi+kLZgUusv1CcjTL76
+ujrHeq+m+BmnijWaZHwhWVANdpm0K0oCiP/m8j6fkbxYS8IYff95SnIFxVIhRSAk
+442NCEdtwwM5iz+XRFq3sKkrfkf+GeJvAoN/kqHuj8dbJBrTO7UT/5wgdEOHmq6n
+en0I1qgsHPn1LZDCwVwEEAEIAAYFAlfNajkACgkQf+o9phacd9YOLQ/9F5fZAMay
+91e7p80UE+6yxX12ImAfN9MqO8yJqB14x8v6lq/Spk04MqsDPwRg7jM6pjTCfnmJ
+j7k2ZaobnKeF+jx6mdF0krpl/ZkR9XUAEOi0rHICFZ8frWyYXluMLq9cBW/d0cI/
+9Y0nBJh6bls7Xt7Si6h8aOnZMtWoWDKeF8Zgll3L1rLaz2oUvHSjVeQrfeGanpde
+ZGe4ygzdW4ztFpJohEVIVV2DX5551uYXzCHGUTGIE0qKLWrjKIR1zN8t2tGUkeEL
+yYWe/+Vng7EDQ4BkFFjQoxyov/X/qPd3e4Ul9Sg++2Oq43Ix5+FEsUeQ1AggPxGG
+XDxB9BfMvDYfWPjSJTZFe5Xvm5kLxBDBS0Ad0Wp1O0ZcV6V5lGWfcE0gpwWlHiKP
+ejqoh3kJb/4HBMPBGXADX5/Er40Rwgf4jLifY/zsKxwQlA+Y1fGJyo6GTuukz7lT
+e1evhCC7ZHPZi5Maeho7HMjNDSNBAznh1k6RnPTl7TMGGM1YCA+tV1kipfLYtWFs
++8o987VjK4Yl/cD5ZsqdcjVN2o9Ju1hwFBCKxU3pFFxz1vHCw8eKiH1MLvCWWlfe
+0HW/oMmbtQPiRwa5MSvOqYLJr5UGND3/69aI/fmLES9um2nwE73+oQvBD09ffM9I
++8Z8ql28HE+dRE7l+XtYG8PPGj9DHFxrn/LCwVwEEAEIAAYFAlfNaMcACgkQLrOE
+AS63E1j8rw//e8qkqc7DmCOXI1wMln0XTkvHGTDxla4GqY+w6uCi5tX7+an9lmXc
+SpPdYzX9EItuXFheoDAYw2SGyyjz9Bm6OFGOTZDLZKAg3S/7vryCBu6vJoFDyRfM
+qlbVkkpUr2E9If5Zj1ibkc0/tFNR3CqdzS63n6REvqo1Qz2uN0pNCxfQVPqgE/LX
+YaIYV0nVobFQxfpa4FqzxS/MsMx9Oxkm2S2QLcaxs6gKoZaV+ki7vSXP+2p+9HTp
+kSVlaNHu5PJqIA0/wq/nmi5iFpO19Nky1TlQ/uY02Lt/aCY20Z3tjb0Bxd19f1ND
+u0GlcTMj8me4i1ZX/LQahm/XWTS1A+Fftdlglodoh7p1otOWg10Iy/yXWl89LOJF
+o2wZqlAnPuIvSiCVerHwM+9fUM843cA2ODP2XPDVg+9ZcA0IyMSyeThpc6gY5BYl
+nn+sSmKApaA5PiAygX5WpgSmcZu2FRgr8pVY6MlpXE0w/f7rUtVlBlpCCZG5h/cO
+4GYzb1sws51ZsJ9tKp+kdsdvAMEJB1aLP4/C3g97OB4vQmDkxaaeHSQNqqivr1HL
+5dvXZ7WfU0jYIw0JLL5DOl0w4bBtrocgc1gaaYVrESs7YuaSGYwUw1w/muOtUOvx
+NckNRimksK+Ab6CcUUlbnTzqtLb2koKIYCSgwTIQpt4GdG4OVLxmAYHCRgQQEQIA
+BgUCV81nSQAKCRD+VES+ZwIWG5RoAJ9deJcaYEHuiYAmhpoAiKM3hG9okQCgi6eB
+GBmK0JN+gnGZQOj+4dG+WA7OwU0EV81kPgEQAL17eQdH3CEDUgBg5ETli9+99/VP
+66bEPCKCzGD6f6/OUXbLVBXs03GzDJzBr6b8ysaWMR7c0WXCKFf7U5+3QI4s9TBl
+csnHxq59343nJh2WsE5cc75WHlfKwse77zT3tujds2MjSBj2324skw9RHSm4mkc7
+esbN8GCQRAM4gUjwEo76SXvVwuUKYaStNLW8DKLaHBxNNclk3v/k7YVmVo07uf/u
+8vIGeUvql/l2XBlWWhxkgaG+UnRwNcpMh0iuVkrJYn04FQtOV20EREVITG1tWHMQ
+4KznqdgdvbEWnFkkbHvj2c6rGylSJShQba9pE1LAbNhF6hbEip2nq6uKsQT0Az2/
+ZWuP83GH4yDHbJCDoT8lf2xcawz+yfd2q9QF8QeIObLCNt09dJNceTbjU3h3BEAv
+D6gGdqzHDPPfmf0UVji1576PEmheMu25/zbh1mvqnsV2YHZsGXVpbBB1PHzrKM1D
+H3PfqFTFQ5elrvkuxhgRrSKaiWBuaGD7DWMiEIV2QKmYutxYS0FJ57lFRaKo0AiV
+vtDLVh0edjkLvkgufxs6oAPnLt5kuo/na/tgWJjY20Eu63dQtIbpPYPrALm2+OMK
+8eAn3tOg7HsUMXQS5hAqRvH/8aMVoW7qjMldgthk5EXTplylkNCCPz9dTqT/g8RB
+mKxelnRfgj3Cxy4dABEBAAHCwXwEGAEIACYCGwwWIQTKJixsg95NL7KKMyo6ak24
+Oeqm1wUCaeaZzwUJFLJaZQAKCRA6ak24Oeqm14VIEACPh699M1V657MXo98Zrafn
+D/LX1tHj1O0R1uTFIvS8DYd/8zseKry3rIKnKFZ+0q+Y7W4ipuTO7iGLzDGXKl8Z
+94A4e+Ov69ADGRDJRxhIjlu7OxSqPJVCcMH1qcOhxEqefcDdHz/xsKEJ49lARs9M
+nCmlnREjVNHLm4n3IOU85qfF5hRIRngIx6ut37F6u+hU36nw6/8ovouKhNUaWL0z
+kGbGvki+Sdz/h8vnkiwEsnUqTJV3QtD4+8teTNqKCUm0zTBWMZLXZcx3uSwDPecE
+nPL90767TDh/vOjgsJcMwElhqhgIksK1KwGgdVGlJ4CcvdxJzkZkpwt93yDJ/v/9
+GDu2GLsKFsL/Ld4JDPHrR4ZGNr2R5fHvjvzMDYeGkq+Ju/KIZfc4eninhvsoSieJ
+yXr2Gx8RQehFt7fVK6eLnJV3kKgz2z+419tJnijSbpw7tUmU18uSiyrZo70beSTn
+pd5I2hAvDd5vyHUAUnIvbW1FQ/TIqSoOK68L+Ifu05b36txe6LfeLSDB1lJa7c1D
+oRxEh2uTTcYiKDBK/txDz/XUAZa0oyZIZ/jOJJ+3KuyVnbBxPZ+tcCVbZN/PCQzT
+wACfe8r7EjgB81yOKxhuIxeDQs5iAP8XAUH4PZTdwOQZ/i6d3xobDjmXfhdsLkRy
+VCTUNQfMjzIj3+7NjMOaoMLBfAQYAQgAJgIbDBYhBMomLGyD3k0vsoozKjpqTbg5
+6qbXBQJnW1/LBQkS0c9lAAoJEDpqTbg56qbXHwEQAIqwM3OfKVFHlzBZJ/ZJ3FPp
+PDk74qchDvKUJqBzZ3kTq/vb82fb4Ji71wyAqpMG8EY8Y8n2k43ffc3yc1RTGLFK
+Het07LCHu1QaJditJE0NPFQZPtIbJycQAEG/5r4ArNKryItAuxypd5Q4sdo8obuA
+av8MCUU1nxYUZXCu6GiwKxyaFQosJcoL4XuuNYmIREgVT+q9gHPPb4aoGJLx+p6c
+Nm/UlvCgoiAd6GOgkqtYJFhelBrYifAWjZPxiCeBN8w61e1rEGi4jiQteDhuIt4x
+mx4yoiahq5UMeZ+buR3YRxpm1MG58ixUJhyQI1IF/ACOAZAgo/2ihk33/GRVpaFa
+v4UOGmTO75328ZOQHs5VGzvzKtXioFpm9FhPLH+tPKj9We9pb3NcNmaNFWPvrYGd
+uvZZKtIYLMliFByNrEHpdPqNxFN7iw48fT/bsBu4y10oQ3LbwJ5LHddV/a/Jyd4F
+cn3cNRPrxGhrGMPDvpk/sBk8lZUM6MxmZI0Cn0bPR0/P7Et6jGJ39PLorY6wmfAH
+9Qk53YkVvKIzZ0prazFy0fyhYSipchCtIBn4XYOdahSsi0+F/BP6INt1MKJPHYBT
+p3JygzmKa4b34wPL0MfOIrd3vu5KG2LZqA3Cg3mWdXdUrxcYYZJ2HdN2BG5vSzls
+kbZiSqGAjKm5+N8EUg6CwsF8BBgBCAAmAhsMFiEEyiYsbIPeTS+yijMqOmpNuDnq
+ptcFAl8gmUkFCQlJgIsACgkQOmpNuDnqptdDfg/+JhP4xHbkk+mwqZwozbDFMp4x
+0wkCe4PrwwV5ZDXBn+uNsG7WjKnj9JdjLCqZsK35QHbOhx2vuw39mbKe38rfLf0b
+r/WvRJkg393kRDW6RLrlUnoyjv3sAA5OV5SkhlRCzwFPILXGyq8yDjKq9xgXuAh/
+N7IaLjYmQf/n07b+tUya+o8ngDguwBl3jFSQyciLWRlf3PAsx9tgkYFMtK4b5KX0
+rcG6RZEVPNJP9XbxPZN4NkBLbmIg0yrb9ISdoPFj7GBtonFk7VPusk0GYtInzKqL
+BozCbj3v4GLTSdEw+bKXyme7xppCTXBNvOyAM2aTBuaLyG6HY6Js5dhme7mPRQwu
+nGSTkmIljoMgm/+r2OUHL3jiV3JO+NAhOfgTDeJ9uLTt3PqfiIsCl2IqaGe80bEs
+ZNyP8963iP9kAcleeNw5WsGZDgR24yKnFFxXAUUBsnOzGbObScaFCh928ZHOM5Dy
+7hFPqvEhVPTTxvEYn4TXKc5Ndq/eY7Mhbw6awEkR0nIfHPW5eeziGZLMHVuldPO4
+Hn9izC6EgX867lPodM5gidzz7A9rm4iGUoeNB9tbfTf18+UiV4ZQ8rEKn+Mr3/2F
+fsEPiBuxS2rnGq5o/Jn+QvWTKxfUPvIBmUMkBGdLhePukcgQ1ExnyKCPeFaniOep
+GApR1pjEW/NwJLyCGUXCwXwEGAEIACYCGwwWIQTKJixsg95NL7KKMyo6ak24Oeqm
+1wUCXVO2UQUJB2eFkwAKCRA6ak24Oeqm13uOEACliNNyVVKeZFrCr5AdgM8CXDoD
+1b4ySuziz9Xj8lolHtxHQu7govXkAeMHNJbrrNDJLCD2LdlndMFHPYpks0nZadqJ
+53oRYfocSrJixU7IgDHOU+epAaCfnzXxawmLiGb+/Hnm+BUH2BcV65ji+HN6SSd8
+ZoC/AfrhXbo0eX0o29GAaLPibZSVlBcdpOvEPLqIr3JLv4S1IMYiz5QA6zPTU4vr
+rB1vHF9laxsqBV8OiSCDWa2cBx2a87h+T8rjyw6JI+hTHOZwlpL40MWFhKke7/iD
+jQ7/AhHY9tuLOAf0DS1xT/Jv0h54Dr6KgW8aBiQ8fW6vwh4+6Tn1EHLq/FhKM9JM
+rt6rb4tZ0ZnzFKcAlkf8N35nDL3DEvgLAukqizknB/qHHMnMn+eswTwgi3AAKU/R
+OXTDjL7PJoqmAVx9wOIik8fTMntJu512f59WPEOhgfu4fEGVDiZXiFVmS9jt0jXv
+qfVI2VGEl7kZZOfeL/xWv9MhrqCOXr6ejmQhItoVOaj+S7eJcSN4uoGGSBh7w1Yj
++ydl3pcYNxOps4/9sBaZgT82VZfkOPcPAvYx3zWvXKkBCimK/VgHLlcriHlz74qT
+n/rkOQ9muRonSS99SxdygxO5MLFvVgDpm7wv+woth0GgdQL/1P1pWa5Vc+z6mdfJ
+M02iWwNgt4zUqk1ePcLBfAQYAQgAJgIbDBYhBMomLGyD3k0vsoozKjpqTbg56qbX
+BQJbYMklBQkFdJhnAAoJEDpqTbg56qbX+6wP/2q6X/0YwEUnTRvyrDkE28e5NkiN
+YeK16QbLvNVxpo5hm8njCg2Nj9svYwGp+WnqqJcysrBA3/++ZPQGfdskqKBjvNG+
+Jsq5dX+jR8zQbJY16/zkFcZDVLP8jEseyr/FUiToxUSzjmTqR8g8ZrNVpDxdEwQ6
+zGEUipxoo8IBQllVZCgl4gzVqlV3AvX+ViR1ZKelICdZ16NeHgY6xFRLc6MBATsk
+XV804uwl0HYeTp65WUjJOg+zQx3b9fLIR02v34jSLkAdFcZ2YverXEJs8kxXUzUL
+u0+qqlzMfvoDf2sTNta91UDntanJKVxNeljWSLRRv1C7u3vPzHviTCY6g3E1dfne
+jr2WjZqsubq8wsdZ8x0uYzkL9usuGqIeikxNpZ3nmEQNRt7emBRoyMomRyvHGke/
+2kSGl1b8Gzg89CALON9ZHBtSNnMLNFQOFF0oqRd+q8tFlNvnbBQGB+t2mVSwpKQk
+odgJAcCirWGs1XI8O5a/wKcYcUIPKHN3adm8QboJso/1JkON5z/yGhF6OsRG69wz
+KZPwaeNcZZswP0mJBLJlFsHfCSIQkTP6SGS5iDYrhcLfTXLCAOZDYVoMLUbQOOuZ
+y2j6Qd61Xyo20u/c6Oi8BK20HlKk7c4Lx72v4Mse85rQidaZR0D8v96MgietW52E
+yJLjetyku17X5QmNwsFlBBgBCAAPBQJXzWQ+AhsMBQkDwmcAAAoJEDpqTbg56qbX
+gxsP/j6dLOSLSsunlKWi2YCIxhFDnb1bI/9rotlRpdQc2S3uKHMB2gKLiIQLFJie
+2XIhkltTkkxkOUMPI0NRAwgtpOEWdgH2CeEvEdqIOI6L/bNPDuqp0yiXwWKWu30q
+9/G+KO3nnLtNtaK8dKSsonrd2VLOS7sPVN7DfzzHpRwJI2TJEd+Q5y905fd8kjP2
+AfVUfOWywXBd03912WIvDF8g+AdRTX/gVgEcsYUIbP0uA8OHLnwvQyo88lF8D3yz
+BDVLVnG7sWnupHX0ldOREHGUqphfnQBTAUpDM/t2lyB/eXev4a14av3mKloBCa8W
+QeecnHVHH59ty+/69dIFTEau2ZEhMUoA+QyXXjbxa1jjiWtjQ4mfzAnwkqChu/Tv
+tCEHzHritSz7I9wBxkxjloejoACylX4Dh5MFYfwhT+fmi0E01FO5UrYT3f48L6sF
+1BWBbwMHYPFY0El9JE+tmSOVGa6noxCVw6bmMgUDeQWVIKsKMt6E8tCrWqN03fUl
+AxKRWim7dWFlpmBy67HFbSOsoiizCLrB1FzGvnQe4ihitQY1NmdjxODsOw16p2hC
+cEjRJYJIX7Z095SKtKhUw0pHhL32YN3zLSKXSImJ4K2aPr8auzTmEOkSx7Qy17YG
+BAma0F+jm5ft431xVW/BLYbZdZoaQAIvlYQvrF4dDTDquKcR
+=wd7s
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/poppler.spec b/poppler.spec
index 8561c7a..a47135b 100644
--- a/poppler.spec
+++ b/poppler.spec
@@ -74,8 +74,9 @@ BuildRequires: cmake(Qt6Widgets)
 BuildRequires: cmake(Qt6Xml)
 %endif
 BuildRequires: boost-devel
-# for %%gpgverify
+# for %%openpgpverify
 BuildRequires: gnupg2
+BuildRequires: openpgpverify
 
 Requires: poppler-data
 
@@ -177,7 +178,7 @@ Command line tools for manipulating PDF files and converting them to
 other formats.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup -p1 -b 3
 
 chmod -x poppler/CairoFontEngine.cc
-- 
2.56.0

